observability/README.md

```diff
@@ -149,14 +150,15 @@ dapr stop --app-id hello-tracing
 
 This quickstart builds on the [distributed calculator](../distributed-calculator/README.md) quickstart and requires Dapr to be installed on a Kubernetes cluster along with a state store. It is suggested to go through the distributed calculator quickstart before this one. If you have not done this then:
 
-1. Clone this repo using `git clone [-b <dapr_version_tag>] https://github.com/dapr/quickstarts.git` and go to the directory named */8.obervability*
+1. Clone this repo using `git clone [-b <dapr_version_tag>] https://github.com/dapr/quickstarts.git` and go to the directory via `cd quickstarts/obervability`.
 2. [Install Dapr on Kubernetes](https://docs.dapr.io/getting-started/install-dapr/#install-dapr-on-a-kubernetes-cluster).
 3. [Configure Redis](https://docs.dapr.io/getting-started/configure-redis/) as a state store for Dapr.
+4. Configure host and password for Redis state store Component in `../distributed-calculator/deploy/redis.yaml`.
```

Comment:
Did you actually need to do this part? I thought we updated that to use the secret store that redis helm install creates by default. I think it just works now?

Reply:
Host might need to be updated. For ARM64, I had to use something else instead of bitnami and defaults did not work. Keeping it there to make sure.

Reply:
Ah. I actually have a hybrid x86/arm64 k8s cluster that I run these on. Probably not a valid expectation for our end users, and it explains why you ran into this.